examples/plotting/plot_rotated_rectangle.py

```diff
@@ -0,0 +1,64 @@
+"""
+====================================
+Drawing a rotated rectangle on a map
+====================================
+
+This example will demonstrate how to draw a rectangle that is rotated relative
+to the axes on a map using :meth:`~sunpy.map.GenericMap.draw_quadrangle`.
+"""
+import matplotlib.pyplot as plt
+
+import astropy.units as u
+from astropy.coordinates import SkyCoord, SkyOffsetFrame
+
+import sunpy.data.sample
+import sunpy.map
+
+################################################################################
+# Let's start with a sample image of AIA 171.
+
+# sphinx_gallery_defer_figures
+
+aia_map = sunpy.map.Map(sunpy.data.sample.AIA_171_IMAGE)
+
+fig = plt.figure()
+ax = fig.add_subplot(projection=aia_map)
+aia_map.plot(axes=ax, clip_interval=(1, 99.99) * u.percent)
+
+################################################################################
+# Define the rotation angle and center coordinate of the rectangle,
+# as well as the width and height in physical units.
+
+# sphinx_gallery_defer_figures
+
+rotation_angle = 90 * u.deg
```

Comment:
Heh, a rotation angle of 90 degrees is one of the least interesting possibilities.
```suggestion
rotation_angle = 30 * u.deg
```